tracing-core/src/callsite.rs

```diff
@@ -185,21 +185,21 @@ mod inner {
     ///
     /// This function is intended for runtime reconfiguration of filters on traces
     /// when the filter recalculation is much less frequent than trace events are.
-    /// The alternative is to have the [`Subscriber`] that supports runtime
+    /// The alternative is to have the [`Collector`] that supports runtime
     /// reconfiguration of filters always return [`Interest::sometimes()`] so that
     /// [`enabled`] is evaluated for every event.
     ///
     /// This function will also re-compute the global maximum level as determined by
-    /// the [`max_level_hint`] method. If a [`Subscriber`]
+    /// the [`max_level_hint`] method. If a [`Collector`]
     /// implementation changes the value returned by its `max_level_hint`
     /// implementation at runtime, then it **must** call this function after that
     /// value changes, in order for the change to be reflected.
     ///
-    /// [`max_level_hint`]: crate::subscriber::Subscriber::max_level_hint
+    /// [`max_level_hint`]: crate::collector::Collector::max_level_hint
     /// [`Callsite`]: crate::callsite::Callsite
-    /// [`enabled`]: crate::subscriber::Subscriber::enabled
-    /// [`Interest::sometimes()`]: crate::subscriber::Interest::sometimes
-    /// [`Subscriber`]: crate::subscriber::Subscriber
+    /// [`enabled`]: crate::collector::Collector::enabled
+    /// [`Interest::sometimes()`]: crate::collector::Interest::sometimes
+    /// [`Collector`]: crate::collector::Collector
```

Comment:
a bunch of references to `Collector` here (and in the link refs, which i think won't work). we should either refer to the trait as "the `Collect`" or just refer to it as lower-case, non-code-formatting "collector" and have it link to the `Collect` trait...